src/mappings/membership/index.ts

```diff
@@ -68,6 +71,11 @@ export async function processMemberProfileUpdatedEvent({
   }
 }
 
+function updateMemberHandle(member: Membership, newHandle: Uint8Array) {
+  member.handleRaw = '0x' + u8aToHex(newHandle)
```

Comment:
The `u8aToHex` function itself prepends the `0x` before the hex string
```suggestion
  member.handleRaw = u8aToHex(newHandle)
```